pilot/cmd/pilot-agent/status/server.go

```diff
@@ -95,6 +101,19 @@ type Server struct {
 	envoyStatsPort      int
 }
 
+func init() {
+	registry := prometheus.NewRegistry()
+	wrapped := prometheus.WrapRegistererWith(map[string]string{"agent": "istio"}, prometheus.Registerer(registry))
+
+	// go collector metrics collide with other metrics.
+	exporter, err := ocprom.NewExporter(ocprom.Options{Registry: registry, Registerer: wrapped, ConstLabels: map[string]string{"agent": "istio"}})
```

Comment:
I think you need to remove ConstLabels here or it will collide with the ones we already pass them from L106?

Reply:
removed